### ISSUE TITLE
Why lr in the Dropout cases is lower?

### ISSUE THREAD

**Issue by justanhduc** [noahfl/sdr-densenet-pytorch#2]
Hi. I went through all the Tensorboard logs and found that the lr in the Dropout cases is smaller than in the proposed method, and this is not mentioned in the paper. Why didn't you train with the same lr? I don't think the benchmark is fair if the lrs are different.

**Comment by noahfl** [comment]
Extremely delayed response, but you're right. The paper was updated with this in mind; both SDR and Dropout are now compared using identical learning rates.
